dts/arm/ti/j721e.dtsi

```diff
@@ -0,0 +1,23 @@
+/* Copyright (C) 2023 BeagleBoard.org Foundation
+ * Copyright (C) 2023 S Prashanth
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <freq.h>
+#include <arm/armv7-r.dtsi>
+/ {
+
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	uart0: uart@2800000 {
```

Comment:
This UART is the same one being used by Linux, ideally this should not be used in R5 firmware, can you use another UART and update?

Reply:
Ok, I will update to UART 1

Reply:
Added Uart 2, as beaglebone ai64 has uart2 pinouts in p8 header.